native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt

```diff
@@ -15,12 +15,37 @@
  */
 package okhttp3
 
+import mockwebserver3.MockResponse
+import mockwebserver3.MockWebServer
 import org.assertj.core.api.AssertionsForClassTypes.assertThat
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
 
 class SampleTest {
+  @JvmField @RegisterExtension val clientRule = OkHttpClientTestRule()
+
   @Test
   fun failingTest() {
     assertThat("hello").isEqualTo("goodbye")
   }
+
+  @Test
+  fun testMockWebServer(server: MockWebServer) {
+    server.enqueue(MockResponse().setBody("abc"))
+
+    val client = clientRule.newClient()
+
+    client.newCall(Request.Builder().url(server.url("/")).build()).execute().use {
+      assertThat(it.body!!.string()).isEqualTo("abc")
+    }
+  }
+
+  @Test
+  fun testExternalSite() {
+    val client = clientRule.newClient()
+
+    client.newCall(Request.Builder().url("https://google.com/robots.txt").build()).execute().use {
```

Comment:
will probably want to get rid of this for running on CI

Reply:
What is the junit5 way of doing this?  I'd like a way to say run this assuming a network and non-hermetic tests allowed.

Reply:
Throw `org.opentest4j.TestAbortedException`, perhaps by calling `Assumptions.assumeTrue(false)`

Reply:
Throw `org.opentest4j.TestAbortedException`, perhaps by calling `org.junit.jupiter.api.Assumptions.assumeTrue(false)`

Reply:
I was thinking https://junit.org/junit5/docs/current/user-guide/#writing-tests-tagging-and-filtering

@Tag("remote")
@Tag("fast")